torch/csrc/distributed/c10d/logging.h

```diff
@@ -10,11 +10,18 @@
 
 #include <c10/util/Logging.h>
 
+namespace torch {
+template <typename... T>
+std::string c10d_format(fmt::string_view fmt, T&&... args) {
+  return fmt::vformat(fmt, fmt::make_format_args(args...));
```

Comment:
I believe you should use `std::forward<Args>(args)...` instead of plain `args` to preserve the value categories of the argument expressions.

Reply:
Forwarding is not needed in `make_format_args`, only in formatting functions.

Reply:
Actually you are right, `make_format_args` could use forwarding here in newer versions of fmt and it won't harm in older, so changed.